.github/workflows/basic.yml

```diff
@@ -19,8 +19,6 @@ jobs:
           #     SC1091: Not following sourced file
           - find tests/CI -name '*.sh' -print0 | xargs -0 -n1 shellcheck --exclude=SC1090,SC1091 --external-source
           - tests/runPylint.sh
-          - tests/py3Check.sh
-          - CHECK=pylintPY3K tests/runPylint.sh
```

Comment:
Then you can remove those sh files too.

Reply:
`tests/runPylint.sh` is still used but I've deleted the `py3Check` files 👍